addons/web/static/src/search/search_model.js

```diff
@@ -175,6 +175,7 @@ export class SearchModel extends EventBus {
         this.fieldService = fieldService;
         this.viewService = view;
         this.dialog = dialog;
+        this.groupBySort = null;
```

Comment:
I would prefer sortByCount, sortByCountASC,...

Reply:
should it be exported in state?

Reply:
maybe add a new param like searchMenuTypes that can be set in view info and would tell the search model to allow sort by count. Use env.config.viewType === 'list' as condition is not really nice as AAB mentioned it